File: segundos 4 lenguajes/js/JugadorPro.js
export class JugadorPro {
    constructor(nombre = "Juan", nivel = 10, copia = null) {
        if (copia) {
            this.nombre = copia.nombre;
            this.nivel = copia.nivel;
        } else {
            this.nombre = nombre;
            this.nivel = nivel;
        }
    }
}

File: segundos 4 lenguajes/c#/ProyectoCsharp/JugadorPro.cs
namespace ProyectoJugadores
{
    public class JugadorPro
    {
        public string Nombre { get; set; }
        public int Nivel { get; set; }

        public JugadorPro()
        {
            Nombre = "Juan";
            Nivel = 10;
        }

        public JugadorPro(string nombre, int nivel)
        {
            Nombre = nombre;
            Nivel = nivel;
        }

        public static JugadorPro Copy(JugadorPro j)
        {
            return new JugadorPro(j.Nombre, j.Nivel);
        }
    }
}
